src/main/kotlin/no/nav/familie/ef/iverksett/oppgave/OppgaveUtil.kt

```diff
@@ -16,6 +17,9 @@ import java.util.Locale
 
 object OppgaveUtil {
 
+    private val fristHovedperiodeForInnhentingAvKarakterutskrift: LocalDate = LocalDate.parse("2023-05-17")
+    private val fristutvidetForInnhentingAvKarakterutskrift = LocalDate.parse("2023-05-18")
```

Comment:
Føler litt at disse  (og funksjonene for utledning av prioritet/frist/beskrivelse) hører hjemme i tasken, eller et sted der vi har annen logikk for karakterutskriftsbrev

Reply:
fixd [1174481](https://github.com/navikt/familie-ef-iverksett/pull/754/commits/1174481e18b968b2bf337bdba40113271ba5f68f)

Reply:
fixd [1174481](https://github.com/navikt/familie-ef-iverksett/pull/754/commits/1174481e18b968b2bf337bdba40113271ba5f68f)